include/revng/TupleTree/Visits.h

```diff
@@ -712,3 +712,12 @@ constexpr bool validateTupleTree(L Check) {
 
   return true;
 }
+
+namespace revng::detail {
```

Comment:
If something it's used outside of the file where it's defined, let's not put it in a `detail` namespace.